lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1246,9 +1246,9 @@ void ResourceMap::loadDefaultConfiguration()
   addAsString("InverseNormalFactory-Method", "MLE", {"MLE", "Moments"});
 
   // KernelMixture parameters //
-  addAsUnsignedInteger("KernelMixture-LargeSize", 20);
+  addAsUnsignedInteger("KernelMixture-LargeSize", 50);
   addAsUnsignedInteger("KernelMixture-PDFCDFDiscretization", 1000);
-  addAsUnsignedInteger("KernelMixture-SmallSize", 50);
+  addAsUnsignedInteger("KernelMixture-SmallSize", 20);
```

Comment:
Warning! It was on purpose that the values were inversed, in order to block the interpolation of the CDF in KernelMixture. The initial idea was to have an automatic interpolation of the CDF by piecewise Hermite polynomials in order to speed-up computations (mainly the iso probabilistic transformation) but is has been proved to be fragile. I was pretty sure it would be easy to fix but it doesn't look so. I should have replaced theses entries by a unique flag, allowing the user to choose if this interpolation is done or not.

Reply:
TODO: add key `KernelMixture-EnableInterpolation=false`

Reply:
Done.